CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed format of `notBefore` and `notAfter` in node and network certificates (#2243).
 - CCF now depends on [Open Enclave 0.14](https://github.com/openenclave/openenclave/releases/tag/v0.14.0).
 
+### Changed
```

Comment:
@letmaik can you create a 0.18.4 section and move this up?

Reply:
done